service/telemetry.go

```diff
@@ -109,8 +110,22 @@ func (tel *colTelemetry) initOnce(col *Collector) error {
 
 	logger.Info("Setting up own telemetry...")
 
-	instanceUUID, _ := uuid.NewRandom()
-	instanceID := instanceUUID.String()
+	if resource == nil {
+		resource = map[string]string{}
+	}
+
+	var instanceID string
+	instanceID = resource[semconv.AttributeServiceInstanceID]
```

Comment:
Allow users to decide to not have this... or have it empty if they want?
```golang
instanceID, ok := resource[semconv.AttributeServiceInstanceID]
if !ok {
}
```

Reply:
Done. You can now provide empty string to suppress built-in attributes.

Reply:
@open-telemetry/collector-approvers I would like opinions on this. Is this a) valuable and b) not-surprising behaviour?

Reply:
Empty strings are valid on resource attributes so it does feel a bit like surprising behavior to me. 

I don't have a good replacement for this though, only thing I can think of is maybe just `service.instance.id: null`? since null attributes are [not valid](https://github.com/open-telemetry/opentelemetry-specification/blob/0c793b450098e8d7a739e575897707f7db08d8ce/specification/common/README.md#attribute).

Reply:
Yeah, `null` is probably a better choice. Let me try that.